router/server.go

```diff
@@ -465,6 +478,27 @@ func (s *Server) runHttpBridgeServers() error {
 	return firstErr
 }
 
+func (s *Server) addPortToConfigService() error {
```

Comment:
This got me thinking. Since other services might also use either the get and post functionality you are building here. Won't it be more efficient to add two methods to the serviceconfigsgo repo to get and set variables. Then all of this is handle for all of the go services at once. What do you think @bmharper